src/server/debugcmd.cc

```diff
@@ -1114,7 +1114,20 @@ void DebugCmd::ObjHist(facade::SinkReplyBuilder* builder) {
 void DebugCmd::Stacktrace(facade::SinkReplyBuilder* builder) {
   fb2::Mutex m;
   shard_set->pool()->AwaitFiberOnAll([&m](unsigned index, ProactorBase* base) {
+    EngineShard* es = EngineShard::tlocal();
+    string txq;
+    if (es) {
+      EngineShard::TxQueueInfo txq_info = es->AnalyzeTxQueue();
```

Comment:
we have this call in several places and the string we create is a littel bit different in the different flows.
Can we make this more generic and return the string from AnalyzeTxQueue?

Reply:
done